roles/kubernetes/preinstall/tasks/0020-verify-settings.yml

```diff
@@ -130,11 +130,11 @@
     - cloud_provider is defined and cloud_provider == "oci"
     - not ignore_assert_errors
 
-- name: Stop if RBAC and anonymous-auth are not enabled when insecure port is disabled
+- name: Stop if RBAC and anonymous-auth are not enabled
   assert:
     that: rbac_enabled and kube_api_anonymous_auth
```

Comment:
Why do we require that ? kube_api_anonymous_auth == true ?

Reply:
hum, legacy thing that can be removed I guess